src/main/java/com/google/api/codegen/transformer/SurfaceTransformerContext.java

```diff
@@ -117,6 +107,13 @@ private boolean isSupported(Method method) {
     return methods;
   }
 
+  private boolean isSupported(Method method) {
+    if (!getFeatureConfig().enableStreaming()) {
+      return !method.getRequestStreaming() && !method.getResponseStreaming();
+    }
+    return true;
+  }
```

Comment:
By looking up the history, I followed the pattern provided by @chrisdunelm here. I am okay with adding `isSupported` into the feature config but I prefer to make the change in a different PR.